docs/contributor/resources/04-watcher.md

```diff
@@ -0,0 +1,11 @@
+# Watcher
+
+The `watchers.operator.kyma-project.io` Custom Resource Definition (CRD) is a comprehensive specification that defines the structure and format used to configure the Watcher resource.
+
+The Watcher custom resource (CR) defines the callback functionality for synchronized remote clusters that allows lower latencies before the Control Plane detects any changes.
```

Comment:
This sentence makes me wonder if we should have a public glossary explaining terms like "synchronized remote cluster", "Control Plane", ...? We can decide on that later, but for now I would propose that we use `Kyma Runtime cluster` and `Kyma Control Plane cluster` instead as those are the terms that we also find in the architecture.md